lms/djangoapps/certificates/views.py

```diff
@@ -591,6 +597,16 @@ def render_html_view(request, user_id, course_id):
     # Append/Override the existing view context values with any course-specific static values from Advanced Settings
     context.update(course.cert_html_view_overrides)
 
+    # track certificate evidence_visited event for analytics when certificate_user and accessing_user are different
+    if request.user and request.user.id != user.id:
+        emit_certificate_event('evidence_visited', user, course_id, course, {
```

Comment:
@mulby @mattdrayer one more question if we already have requesting user in the context then can we remove `accessing_user_id` field from event data of `edx.certificate.evidence_visited` event?